instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/RuntimeConfig.kt

```diff
@@ -92,7 +92,7 @@ internal object RuntimeConfig {
 
     val stringAttributes = mapOf(
         "version_name" to BuildConfig.VERSION_NAME,
-        "device" to Build.DEVICE,
+        "build_device" to Build.DEVICE,
```

Comment:
should we revert this change now?

Reply:
Yes, good catch. I actually thought I did revert it.